mesh/accountsettings/accountsettings.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE IF NOT EXISTS AccountSettings(
+	settingID INT PRIMARY KEY AUTO_INCREMENT,
```

Comment:
1. Do we need a unique ID for settings other than the `FOREIGN KEY accountID`? I believe this would only be useful if we wanted to be sharing a set of settings amongst several accounts/users, which I don't believe to be the intention here. 
1. nitpick: there's an extra indentation here

Reply:
@rezatnoMsirhC I do agree with you here, so we can identify settings just plainly on foreign key in the table.